ir/src/constraints/graph.rs

```diff
@@ -348,6 +353,9 @@ impl AlgebraicGraph {
             IdentifierType::Constant(ConstantType::Scalar(_)) => {
                 self.insert_constant(ConstantValue::Scalar(ident.to_string()), domain)
             }
+            IdentifierType::RandomValue(offset, _size) => {
```

Comment:
minor nit: @grjte pointed this out elsewhere that it would be better to do RandomValue(offset, _)